### PR TITLE
Allow AOs to create their organisation

### PR DESCRIPTION
**Pull request by l-vincent-l**
If you don't have an organisation yet when you click on "Open my data" you'll be directly redirected to the organisation form.
If you already have an organisation you'll see a button above the organisation list
image

The organisation form : 

image

partial_fix #44

### CODE REVIEW

**Comment by maukoquiroga** [comment]
Could you add the corresponding issue ?

File: lib/transport_web/controllers/user_controller.ex [hunk 1]
```diff
@@ -21,6 +23,41 @@ defmodule TransportWeb.UserController do
     end
   end
 
+  def organization_form(conn, _params) do
```
Comment (maukoquiroga) [inline lib/transport_web/controllers/user_controller.ex]: Wouldn't it be clearer a ```TransportWeb.User.OrganizationController``` ? That way controller action name will simply be ```new```, which feels more natural than ```form```.

File: test/transport_web/controllers/user_controller_test.exs [hunk 2]
```diff
@@ -65,4 +94,70 @@ defmodule TransportWeb.UserControllerTest do
...
+      assert redirected_to(conn, 302) == user_path(conn, :organization_form)
+
+      conn = conn
+      |> get(user_path(conn, :organization_form))
+      assert false == (html_response(conn, 200) =~ dgettext("user", "You need to provide an organisation name"))
+      assert html_response(conn, 200) =~ dgettext("user", "You need to provide an organisation description")
+    end
+
+    test "name field is empty", %{conn: conn} do
+      conn = conn
```
Comment (maukoquiroga) [inline test/transport_web/controllers/user_controller_test.exs]: https://github.com/christopheradams/elixir_style_guide#pipe-operator

File: lib/transport_web/controllers/user_controller.ex [hunk 3]
```diff
@@ -21,6 +23,41 @@ defmodule TransportWeb.UserController do
...
+    render conn, "organization_form.html"
+  end
+
+  def organization_create(conn, %{"name" => name,
```
Comment (maukoquiroga) [inline lib/transport_web/controllers/user_controller.ex]: Validation logic shouldn't be controller's responsibility. That should probably go into a struct or another data structure.

**Comment by maukoquiroga** [comment]
Looks good, a couple of comments.

File: lib/transport_web/controllers/user_controller.ex (hunk 1 above)
Comment (l-vincent-l) [reply lib/transport_web/controllers/user_controller.ex]: Yep, I'll open an issue for that.
This PR is already too big.

Comment (maukoquiroga) [reply lib/transport_web/controllers/user_controller.ex]: OK

**Pull request merged**